PCbuild/rmpyc.py

```diff
@@ -1,25 +1,23 @@
-# Remove all the .pyc and .pyo files under ../Lib.
+# Remove all the .pyc files under ../Lib.
 
 
 def deltree(root):
     import os
     from os.path import join
 
-    npyc = npyo = 0
+    npyc = 0
     for root, dirs, files in os.walk(root):
         for name in files:
             delete = False
-            if name.endswith('.pyc'):
+            # to be thorough
+            if name.endswith(('.pyc', '.pyo')):
                 delete = True
                 npyc += 1
-            elif name.endswith('.pyo'):
-                delete = True
-                npyo += 1
 
             if delete:
                 os.remove(join(root, name))
```

Comment:
At this point you're free to dispose of the `delete` var and just do the delete in the `if` branch.